image/templates/helm/stackrox-secured-cluster/values-scanner-v4-indexer.yaml.example

```diff
@@ -0,0 +1,100 @@
+# # NOTE:
+# # The Scanner v4 Indexer is not available in the secured clusters installed via roxctl.
+#
+# # Public configuration options for the StackRox Scanner v4 Indexer:
+# # To run Scanner v4 in the secured cluster, you must connect the Scanner v4 to Sensor.
+#
+# # WARNING:
+# #
+# # Installing in the same namespace as Central is unsupported. In that environment,
+# # Sensor will use the existing Scanner v4 Indexer deployed via CentralServices to scan cluster-local and delegated images.
+#
+# Image configuration for scanner v4:
+# # For a complete example, see the `values-public.yaml.example` file.
+# image:
+#  # Configuration for the `scanner-v4` image that is used by Scanner v4 Indexer.
+#  scanner-v4:
+#    indexer:
+#      registry: null
+#      name: scanner-slim
+#      tag: null
+#      fullRef: null
+#
+# scanner-v4:
```

Comment:
For central helm chart we were using scannerV4 in the values file, and I think even in this PR we have different namings, some using kebab case some using camel case, I see the explanation above on @RTann, but in this case camel case really seems to be more consistent and not required for "helm magic" to work.

Reply:
yes, this was simply wrong. In the code the kebabCase identifier were actually used (https://github.com/stackrox/stackrox/blob/d28a7cfcc5608335f4a01bae8506dc3a6233f7bb/image/templates/helm/stackrox-secured-cluster/templates/_init.tpl.htpl#L295).